templates/core/terraform/airlock/variables.tf

```diff
@@ -25,3 +25,9 @@ variable "mgmt_acr_name" {
   type        = string
   description = "Management ACR name"
 }
+
+variable "arm_subscription_id" {
+  description = "The subscription id to create the resource processor permission/role. If not supplied will use the TF context."
```

Comment:
Doesn't look like you use it for permission/role. Is the description up to date?

Reply:
fixed